sdk/personalizer/Azure.AI.Personalizer/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0012:Avoid single word type names", Justification = "<Pending>", Scope = "type", Target = "~T:Azure.AI.Personalizer.Models.Evaluation")]
```

Comment:
A few things -
1. It is preferred to suppress these in code.
2. If you don't fill in the `Justification` field with something other than the default, a stricter build system will complain.
3. Are you sure these cannot be fixed, for example by using properties? Since this is a fresh SDK, it would be nice to keep things clean.

Reply:
The auto-rest generated a client for each endpoint we had. These suppressions are from a wrapper client class that I wrote which instantiates all the clients. My goal is to have the wrapped clients' methods callable through the wrapper client in the following way. 

wrapperClient.WrappedClient.MethodName()... I do not know how to do this without having the fields of the wrapper client public.

Do you have any suggestions?